azurelinuxagent/common/event.py

```diff
@@ -71,11 +71,11 @@ def save_event(self, data):
         except IOError as e:
             raise EventError("Failed to write events to file:{0}", e)
 
-    def add_event(self, name, op="", is_success=True, duration=0, version="1.0",
+    def add_event(self, name, op="", is_success=True, duration=0, version=AGENT_VERSION,
```

Comment:
I think version should be a required argument. In the past we've had issues with people calling add_event without paying close attention. So making the default AGENT_VERSION may actually be causing some extension telemetry to be reported with agent version.